perfdash/deploy.sh

```diff
@@ -14,14 +14,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -euxo pipefail
+set -euo pipefail
+set -x
```

Comment:
Could you explain why you're making this change (I'm curious what's the difference)

Reply:
`set -euo pipefail` is the usual clause put at the top of all bash scripts, to have sane behavior and I as is is easily recognizable. `-x` is to make the script verbose and you can see it much less often that the former. I like to have it on a separate line, so I am not surprised, when the script starts to spit many lines. 

This might be personal I'm fine with reverting it.

Reply:
This is fine. Thanks for explaining

/lgtm
/approve